definitions/json/CaseField/CaseField-WA.json

```diff
@@ -85,6 +85,13 @@
     "FieldType": "etICHearingNotListedUDLHearing",
     "SecurityClassification": "Public"
   },
+  {
+    "CaseTypeID": "ET_EnglandWales",
+    "ID": "etICHearingNotListedDoNotListHearing",
+    "Label": "Do not list",
+    "FieldType": "etICHearingNotListedDoNotListHearing",
```

Comment:
Is this field type correct? PR doesn't introduce a complex type and I can't seem to find one matching this type?

Reply:
corrected